setup.cfg

```diff
@@ -33,7 +33,7 @@ python_requires = >=3.7,<3.12
 packages = find:
 zip_safe = False
 install_requires =
-    pydantic>=1.9.1,<1.10  ; python_version<"3.11"
+    pydantic>=1.9.1 ; python_version<"3.11"
```

Comment:
```suggestion
    pydantic>=1.9.1,!=1.10.0 ; python_version<"3.11"
```

Reply:
the breaking change in 1.10.0 are there in 1.10.1 and 1.10.2. So this restriction does not address that. But it seems the pygls is not using the APIs that actually were broken by 1.10.0 from what I can tell.

Reply:
That's right. Pygls isn't using those APIs that broke in Pydantic v1.10.0, and that's proved by our test suite matrix running against Python v3.11rc2. But I'm not clear how we prove that there aren't Pygls custom server developers out there that _are_ depending on those Pydantic APIs? As far as I can see there's no way to prove that, as unlikely as it might be.

Reply:
> I'm not clear how we prove that there aren't Pygls custom server developers out there that _are_ depending on those Pydantic APIs?

Does it matter? As in if someone used those versions of Pydantic and used those broken APIs would it break pygls itself, or just their own code? If it's the latter then I would argue that's their business and not pygls'; Python is a language for consenting adults and if someone wants to use a version of pydantic with some bugs that's their choice.